contracts/Strategy.sol

```diff
@@ -557,10 +572,7 @@ contract Strategy is BaseStrategy {
         if (pid == 0) return;
 
         uint256 fractionsToStake = kashiFractionInPair(kashiPair);
-
-        if (fractionsToStake > dustThreshold) {
```

Comment:
why did you remove the threshold here?

Reply:
This was preventing us from claim sushi rewards when the fraction was set to 0.